Gemfile

```diff
@@ -1,5 +1,7 @@
+# frozen_string_literal: true
```

Comment:
Didn't you include the configuration for rubocop so that you don't have to do this evey time?